examples/WebRadioDemo/WebRadioDemo.ino

```diff
@@ -0,0 +1,113 @@
+/**
+  Wiring:
+  --------------------------------
+  | VS1053  | ESP8266 |  Other   |
+  --------------------------------
+  |   SCK   |   D5    |    -     |
+  |   MISO  |   D6    |    -     |
+  |   MOSI  |   D7    |    -     |
+  |   XRST  |    -    |    -     |
+  |   CS    |   D1    |    -     |
+  |   DCS   |   D0    |    -     |
+  |   DREQ  |   D3    |    -     |
+  |   5V    |    -    |   VCC    |
+  |   GND   |    -    |   GND    |
+  --------------------------------
+
+  Dependencies:
+  -VS1053 library by baldram (https://github.com/baldram/ESP_VS1053_Library)
+  -ESP8266Wifi
+
+  Instructions:
+  -Build the hardware
+  -Set the station in this file
+  -Upload the program
+
+  IDE Settings (Tools):
+  -IwIP Variant: v1.4 Higher Bandwidth
+  -CPU Frequency: 160Hz
+*/
+
+#include <VS1053.h>
+#include <ESP8266WiFi.h>
+
+#define VS1053_CS     D1
+#define VS1053_DCS    D0
+#define VS1053_DREQ   D3
+
+// Default volume
+#define VOLUME  80
+
+VS1053 player(VS1053_CS, VS1053_DCS, VS1053_DREQ);
+WiFiClient client;
+
+// WiFi settings example, substitute your own
+char* ssid = "TP-Link";
+const char* password = "xxxxxxxx";
+     
+//  http://comet.shoutca.st:8563/1
+const char *host = "comet.shoutca.st";
+const char *path = "/1";
+int httpPort = 8563;
+
+uint8_t mp3buff[32];
+
```

Comment:
I like it. Clean and simple example. With many useful comments explaining the thing. Thanks!